tests/scripts/pulp_file/test_distribution.sh

```diff
@@ -37,6 +38,12 @@ expect_succ pulp file distribution update \
   --base-path "cli_test_file_distro" \
   --publication "$PUBLICATION_HREF"
 
+
+expect_succ pulp file distribution show --base-path "cli_test_file_distro"
+test "$(echo "$OUTPUT" | jq -r .pulp_href)" = "$DISTRIBUTION_HREF"
+expect_succ pulp file distribution list --base-path "cli"
+test "$(echo "$OUTPUT" | jq -r .[0].pulp_href)" = "$DISTRIBUTION_HREF"
```

Comment:
This seems very brittle--I often run this test against my dev server which may have other file distributions. Maybe we could just check that the number of results is > 0?

Reply:
If the filter `--base-path "cli"` is working correctly, there must only be a single item returned. So we could even check for cout==1, no?

Reply:
That's true altough the filter is `base_path__icontains` though so there's still a chance.